src/data/taxonomy-redirects.json

```diff
@@ -832,14 +840,15 @@
     ]
   },
   {
-    "url": "/docs/infrastructure/install-infrastructure-agent/linux-installation",
+    "url": "https://developer.newrelic.com/instant-observability/?search=linux",
```

Comment:
@paperclypse, like above, why are we sending users to the developer site about Linux?